pkg/flare/strip.go

```diff
@@ -27,43 +27,35 @@ var blankRegex = regexp.MustCompile(`^\s*$`)
 var replacers []replacer
 
 func init() {
-	apiKeyReplacer = replacer{
-		regex: regexp.MustCompile(`(?m)^\s*api_key:( *['"]?\w+['"]? ?,?)+$`),
-		replFunc: func(b []byte) []byte {
-			s := string(b)
-			s = strings.TrimSuffix(s, `"`)
-			s = strings.TrimSuffix(s, `'`)
-			replacement := "api_key: **************************" + s[len(s)-5:]
-			return []byte(replacement)
-		},
-	}
-	dockerAPIKeyReplacer = replacer{
-		regex: regexp.MustCompile(`DD_API_KEY=['"]?\w+['"]?`),
-		replFunc: func(b []byte) []byte {
-			s := string(b)
-			s = strings.TrimSuffix(s, `"`)
-			s = strings.TrimSuffix(s, `'`)
-			replacement := "DD_API_KEY=**************************" + s[len(s)-5:]
-			return []byte(replacement)
-		},
+	apiKeyReplacer := replacer{
```

Comment:
wouldn't this replace every string longer than 32 characters ?

Reply:
Every hexadecimal 32 character long string yes.